packages/i18n/src/build.ts

```diff
@@ -166,6 +166,11 @@ function _parseMessagesObject(
       ];
     }
     case 'object':
+      if ([null, undefined].includes(object)) {
+        throw new Error(
+          `While parsing the @wxt-dev/i18n package has found that one of the values in one of your translation files is null or undefined. You may have wanted this value to be transalted from the original please test as necessary`,
+        );
```

Comment:
Only the source language should be parsed by the module. That means this isn't a problem with your translation service missing a translation, but a formatting problem with a file maintained by a human. I think we should use a simpler error message that indicates that:

```suggestion
        throw new Error(`Messages file should not contain \`${object}\` (found at "${path.join('.')}"`);
```

Reply:
Added the test and updated the error message